docs/server-behavior.md

```diff
@@ -56,6 +56,12 @@ Applications should generally treat `HEAD` requests in the same manner as `GET`
 
 One exception to this might be if your application serves large file downloads, in which case you might wish to only generate the response headers.
 
+### Zero copy send
+
+Uvicorn supports [ASGI Zero Copy Send Extension](https://asgi.readthedocs.io/en/latest/extensions.html#zero-copy-send). Using the method specified by the extension, you can call the zero-copy interface of the operating system to send files.
+
+Note: This is currently only available on Python3.7+ and is temporarily unavailable with uvloop.
```

Comment:
could we link to a uvloop issue that tracks sendfile implementation ?

Reply:
https://github.com/encode/uvicorn/pull/1210/commits/9971c6bc48254018db55f17a25eefabf313831cd done.